pkg/allocation/converters/converter_test.go

```diff
@@ -647,15 +653,17 @@ func TestConvertAllocationResponseToGSA(t *testing.T) {
 		{
 			name: "Empty fields",
 			in: &pb.AllocationResponse{
-				Ports: []*pb.AllocationResponse_GameServerStatusPort{},
+				Ports:  []*pb.AllocationResponse_GameServerStatusPort{},
+				Source: "33.188.237.156:443",
 			},
 			want: &allocationv1.GameServerAllocation{
 				TypeMeta: metav1.TypeMeta{
 					Kind:       "GameServerAllocation",
 					APIVersion: "allocation.agones.dev/v1",
 				},
 				Status: allocationv1.GameServerAllocationStatus{
-					State: allocationv1.GameServerAllocationAllocated,
+					State:  allocationv1.GameServerAllocationAllocated,
+					Source: "33.188.237.156:443",
```

Comment:
If this is a real IP, it could leave you open to attack

Reply:
Thanks for the remind. It's a fake one.

Reply:
optional: https://datatracker.ietf.org/doc/html/rfc5737#section-3 if you want a test IP.